thrift/history.thrift

```diff
@@ -508,6 +513,7 @@ service HistoryService {
       5: shared.DomainNotActiveError domainNotActiveError,
       6: shared.LimitExceededError limitExceededError,
       7: shared.ServiceBusyError serviceBusyError,
+      8: shared.WorkflowExecutionAlreadyCompletedError workflowExecutionAlreadyCompletedError,
```

Comment:
This will cause some issues at client side I think. For record heartbeat, right now client side will check if the error type is entity not exist and cancel the activity if so.

Reply:
I will send a client diff to handle this in `internal/internal_task_handlers.go:internalHeartBeat`